jobs/e2e_node/image-config-canary.yaml

```diff
@@ -0,0 +1,9 @@
+images:
+  ubuntu:
+    image_family: pipeline-1-28
+    project: ubuntu-os-gke-cloud
+    metadata: "user-data</workspace/test-infra/jobs/e2e_node/containerd/init.yaml,cni-template</workspace/test-infra/jobs/e2e_node/containerd/cni.template,containerd-config</workspace/test-infra/jobs/e2e_node/containerd/config.toml"
+  cos-stable:
+    image_family: cos-beta
```

Comment:
Let's us do `cos-97-lts` not `cos-beta`. (going from the comment that we are deleting). See what image exactly gets picked up today for both options:

```
davanum@cloudshell:~ (k8s-staging-sig-docs)$ gcloud compute images describe-from-family cos-97-lts --project cos-cloud --format='value(name)'
cos-97-16919-353-4
davanum@cloudshell:~ (k8s-staging-sig-docs)$ gcloud compute images describe-from-family cos-beta --project cos-cloud --format='value(name)'
cos-beta-105-17412-156-5
```

Reply:
done

Reply:
it works even better with these images:
```
Summarizing 2 Failures:
  [FAIL] [sig-node] Summary API [NodeConformance] when querying /stats/summary [It] should report resource usage through the stats api
  test/e2e_node/summary_test.go:336
  [TIMEDOUT] [sig-node] Pods [It] should cap back-off at MaxContainerBackOff [Slow][NodeConformance]
  test/e2e/common/node/pods.go:718

Ran 121 of 543 Specs in 3603.273 seconds
FAIL! - Suite Timeout Elapsed -- 119 Passed | 2 Failed | 0 Pending | 422 Skipped
```